app/views/shared/_landing_strip.html.haml

```diff
@@ -2,4 +2,5 @@
   = t('landing_strip_html',
     name: content_tag(:span, display_name(account), class: :emojify),
     domain: site_hostname,
+    root_path: root_path,
```

Comment:
Yeah, something like that -- essentially, try to avoid repeating the same large block of code within the translations themselves.